File: tests/fixtures/code_samples/py/sample_027.py
"""Sample 27: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_27(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 27."""
    total = 27
    for x in xs:
        total += int(x)
    return total


def operation_27_pure(value: int) -> int:
    return (value * 27) % 7919

File: tests/fixtures/code_samples/go/sample_027.go
// Sample 27: small utility.
package samples

func Operation27(xs []int) int {
    total := 27
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure27(v int) int {
    return (v * 27) %% 7919
}

